src/client/vite.config.ts

```diff
@@ -153,62 +156,50 @@ const customPreloadPlugin = () => {
   return result
 }
 
-const copyOpenfinPlugin = (dev: boolean) => {
+const copyPlugin = (
+  isDev: boolean,
+  buildTarget: "web" | "openfin" | "launcher",
```

Comment:
1. "launcher" shouldn't be a thing anymore, right?
1. type `BuildTarget` is defined on L17

Reply:
dang .. well spotted

Reply:
sorted now, and I added `"finsemble"` as that is a genuine build target that I had not included (out of sight .. eh?)
but haven't used it for anything .. previously it seemed to just inherit the general stuff, including a bunch of web artefacts that I am sure it doesn't really need .. but I cannot test it easily right now ... so if we need to boot up Finsemble in future, we will need to verify some of this stuff, and possibly code-switching points (do we need any more or any less .finsemble.ts alternative implementations .. what about latest FDC3 etc. etc.
cc: @Clark-Adaptive